web/core/templates/index.html

```diff
@@ -66,6 +66,28 @@ <h3>ALL SHOWS</h3>
     <input type="submit" value="Add Text Show">
 
   </form>
+
+  <hr style="margin-top:30px;">
+  <hr>
+  <hr>
+
+  add a WASM show below! just write some WASM!!
+
+  <i>HOW TO WASM:</i>
+  <pre>
+    - Paste your <a href="https://developer.mozilla.org/en-US/docs/WebAssembly/Understanding_the_text_format">WebAssembly Text (WAT)</a> file here
+    - have a `next_frame` function that returns a 96x38 array of booleans, or an empty array to indicate you're done
+  </pre>
+
+  feel free to <a href="https://recurse.zulipchat.com/#narrow/dm/7864">ping @jessechen</a> or <a href="https://github.com/jessechen/bubbletea">make a pull request</a> to add more!!
```

Comment:
extremely small comment, this block (re: "ping Jesse") is related to the "add text show" above, if you want to move that above

Reply:
Indeed! Mostly a copy paste error, will definitely fix